tests/cdc-bank/bank.go

```diff
@@ -345,7 +345,11 @@ func getDDLEndTs(db *sql.DB, tableName string) (result string, err error) {
 			return "", err
 		}
 		if line.JobType == "create table" && line.TblName == tableName && line.State == "synced" {
-			return line.EndTime, nil
+			if line.EndTime == nil {
+				log.Warnf("ddl end time is null, line=%+v", line)
+				continue
```

Comment:
It's not right to continue here, if so, `getDDLEndTs` finally returns error(cannot find in ddl history, tableName ... ). I mean, the caller of `getDDLEndTs` need to handle the situation where the result is an empty string.

Reply:
if meet empty end time, just ignore that round